src/components/ActionsMenu/ActionsMenu.st.css

```diff
@@ -0,0 +1,65 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback;
+}
+:import {
+    -st-from: "./Item/ActionsMenuItem.st.css";
+    -st-default: ActionMenuItem;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultMenuBackgroundColor: color-1;
+    DefaultMenuBorderWidth: 1px;
+    DefaultMenuBorderColor: rgba(255, 255, 255, 0.4);
+}
+
+:vars {
+    /*Overrides*/
+    MenuBackgroundColor:  --overridable;
+    MenuBorderWidth:  --overridable;
+    MenuBorderColor:  --overridable;
+}
+
+:vars {
+    menuBackgroundColor: color(fallback(value(MenuBackgroundColor), value(DefaultMenuBackgroundColor)));
+    menuBorderWidth: fallback(value(MenuBorderWidth), value(DefaultMenuBorderWidth));
+    menuBorderColor: color(fallback(value(MenuBorderColor), value(DefaultMenuBorderColor)));
+}
+
+.root {
+    -st-states: mobile, alignment(enum(left, center, right));
+    padding: 12px 0;
+    max-width: 320px;
+    background-color: value(menuBackgroundColor);
+    border-style: solid;
+    border-width: value(menuBorderWidth);
+    border-color: value(menuBorderColor);
+}
+
+.item {
+    -st-extends: ActionMenuItem;
```

Comment:
I don't think this is needed if we're not overriding anything here.

Reply:
without it alignment would not work